app/controllers/advices_controller.rb

```diff
@@ -3,7 +3,7 @@ class AdvicesController < ApplicationController
   before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
 
   def index
-    @categories = Category.all.order("name")
+    @categories = Category.all.order({ name: :asc })
```

Comment:
伝え方が悪かったかもしれないのですが、下記の様な記述方法で良さそうという感じでした
```suggestion
    @categories = Category.all.order(:name)
```